# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/shadow/stdint-uintmax.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

find_new_sub_{u,g}ids.c cast uid_t to uintmax_t for the "%ju" diagnostics
but never include <stdint.h>, which fails to build wherever the type is not
pulled in indirectly (seen cross building for ia64):

  find_new_sub_uids.c:80:27: error: 'uintmax_t' undeclared (first use in this function)

Signed-off-by: René Rebe <rene@exactco.de>

--- a/lib/find_new_sub_uids.c
+++ b/lib/find_new_sub_uids.c
@@ -9,6 +10,7 @@
 #ifdef ENABLE_SUBIDS
 
 #include <stdio.h>
+#include <stdint.h>
 #include <errno.h>
 
 #include "prototypes.h"
--- a/lib/find_new_sub_gids.c
+++ b/lib/find_new_sub_gids.c
@@ -9,6 +10,7 @@
 #ifdef ENABLE_SUBIDS
 
 #include <stdio.h>
+#include <stdint.h>
 #include <errno.h>
 
 #include "prototypes.h"
